__tests__/app.test.js

```diff
@@ -0,0 +1,64 @@
+const {topicData, articleData, userData, commentData} = require("../db/data/test-data/index")
+const request = require ("supertest")
+const app = require("../app")
+const connection = require("../db/connection")
+const seed = require("../db/seeds/seed")
+
+
+beforeEach(()=>{
+return seed({topicData, articleData, userData, commentData})
+})
+
+afterAll(()=>{connection.end})
+
+describe('GET /api/topics', () => {
+    it('200: should respond with a 200 status code ', () => {
+        return request(app).get("/api/topics").expect(200)
+    });
+
+    it("200: should return an array", () => {
+        return request(app)
+          .get("/api/topics")
+          .then((data) => {
+            expect(Array.isArray(data.body)).toBe(true);
+          });
+      });
+
+      it("200 should respond with an array of all topics", () => {
+        return request(app)
+          .get("/api/topics")
+          .expect(200)
+          .then(({ body }) => {
+            const topics = body;
+    
+            expect(topics).toHaveLength(3);
+
+            topics.forEach((topic) => {
+              expect(topic).toMatchObject({
+                description: expect.any(String),
+                slug: expect.any(String),
```

Comment:
Generally, I like the way you built up these tests, but it's also okay to write them as a single test :)
Also well done for checking the length!